zerver/tests/test_message_send.py

```diff
@@ -1885,6 +1885,18 @@ def test_basic_check_message_call(self) -> None:
         ret = check_message(sender, client, addressee, message_content)
         self.assertEqual(ret['message'].sender.id, sender.id)
 
+    def test_guest_user_can_send_message(self) -> None:
+        # Guest users can write to web_public streams.
+        sender = self.example_user("polonius")
+        client = make_client(name="test suite")
+        stream_name = "Web Public Stream"
```

Comment:
Can we just use Rome here, with an assertion that Polonium isn't subscribed and that it's web_public, rather than making a new stream?